Ontology/Capability/Sensor/WaterLevelSensor/WaterLevelSensor.json

```diff
@@ -0,0 +1,11 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:core:WaterLevelSensor;1",
```

Comment:
I would suggest this belongs in a sub-class instead of directly extending Sensor because Water is a substance and we don't directly inherit substances from Sensor.

Suggest introducing a HeightSensor-->WaterHeightSensor

Reply:
Done. I've called it LevelSensor as many experts are calling Liquid Level Sensor or Water Level Sensor.
I've created LevelSensor->WaterLevelSensor